index.html

```diff
@@ -53,7 +53,7 @@
                   placeholder="Пошук по магазину"
                 />
               </div>
-              <div class="header__language-selector language-selector">
+              <div class="header__language-selector">
```

Comment:
Why you decided to remove class `language-selector`?  It's part of the BEM. 
On line 57 you use `language-selector__button` which is an element but there is no block for it

Reply:
Sorry, it's my fault.

Reply:
It's ok. Do treat it so seriously :)

Reply:
Okay)